packages/node_modules/@ciscospark/widget-space/src/container.js

```diff
@@ -193,7 +195,30 @@ export class SpaceWidget extends Component {
     } = props;
 
     const {formatMessage} = props.intl;
-    if (sparkInstance && spaceDetails) {
+    if (widgetStatus.get(`isError`)) {
+      const error = widgetSpace.get(`error`);
+      const errorTitle = formatMessage(messages.unableToLoad);
+      let errorSubtitle;
+      switch (error.name) {
+      case `Bad Space Id`:
+        errorSubtitle = formatMessage(messages.badSpaceId);
+        break;
+      case `NotFound`:
+        errorSubtitle = formatMessage(messages.errorNotFound);
+        break;
+      default:
+        errorSubtitle = formatMessage(messages.unknownError);
```

Comment:
These were the only errors I could get and since there is no official documentation on the errors, we're stuck with this. Would be good to put some sort of metrics or logging on any errors that don't match.